scripts/create-next-modules.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Description: Create a tarball of the node_modules directory from clean next.js install
+# for the next.js standalone build. This needs to be done because the next.js standalone
+# build for monorepo does not include all the files in node_modules directory in the output.
+# Usage: ./scripts/create-next-modules.sh
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd /tmp
+# Create a clean next.js install using the same version as the current project
+npx -y create-next-app@12.3.1 --typescript --use-npm next_standalone
```

Comment:
current next version could be read from the package.json file? At least node has such helper packages. Not sure how easy it is to do with bash. Now when the version is hardcoded, it might be a hard to remember to maintain

Reply:
Maybe something like this: https://unix.stackexchange.com/questions/243428/how-to-extract-data-from-a-json-file

Reply:
I was thinking of that, but left it out since this script is in the root, which doesn't have next defined in package.json; so it would need to get it from one of the apps then... Maybe it's not too bad, since they would be most likely all in same next version anyway - but I left it out because of that. Another option would be to add the version as param to the script.

Reply:
Where/when this script is called? If it's from the dockerfile then it knows the path would be `apps/${PROJECT}/package.json` where `PROJECT` is the docker build arg.

Reply:
But perhaps first it's better to be sure that this whole approach is correct and working in every environment before starting optimizing it :)